### PR TITLE
Python notebook for training on Common Voice

### PR DESCRIPTION
**Pull request by JRMeyer**

### CODE REVIEW

**Comment by JRMeyer** [comment]
I'm looking into how to integrate the notebooks with our CI, as mentioned here: https://github.com/coqui-ai/STT/pull/1965#issuecomment-918240568

File: .github/workflows/build-and-test.yml [hunk 1]
```diff
@@ -695,6 +695,11 @@ jobs:
           mv convert_graphdef_memmapped_format /tmp
       - run: |
           mkdir -p ${CI_ARTIFACTS_DIR} || true
+      - name: Run python notebooks
+        run: |
+          sudo apt-get install -y --no-install-recommends libopusfile0 libopus-dev libopusfile-dev
+          python -m pip install jupyter
+          ./ci_scripts/notebook-tests.sh
```
Comment (reuben) [inline .github/workflows/build-and-test.yml:702]: Don't add this to the basic training tests, it'll slow them down and they have downstream jobs that depend on them. Add a separate job for the notebook tests.

**Review by reuben — APPROVED** [review]
Just a small comment.

File: .github/workflows/build-and-test.yml (hunk 1 above)
Comment (JRMeyer) [reply .github/workflows/build-and-test.yml]: Addressed in c78f98a

**Review by reuben — APPROVED** [review]
🚀

**Pull request merged**